### PR TITLE
Feature/add support for ma jobs

### PR DESCRIPTION
**Pull request by UntappedSolutions**

### CODE REVIEW

File: action-processor/create-action.js [hunk 1]
```diff
@@ -444,6 +448,26 @@ async function createMarketingAuthorisationRenewalJob (featureName, action){
...
+    log.info(`${SERVICE_NAME}::createMarketingAuthorisationNewJob::${action.label}::updated:${JSON.stringify(responseData)}`)
+    var savedAction = localStorage.getItem(featureName, action.label)
+    savedAction.response = responseData
+    log.debug(`${SERVICE_NAME}::createMarketingAuthorisationNewJob, saved action ${JSON.stringify(savedAction)}`)
+    localStorage.setItem(featureName, action.label, savedAction)
+    var maNewJobList = localStorage.getItem(featureName, 'maNewJobList')
+    if (!maNewJobList) {
+        maNewJobList = []
+    }
+    maNewJobList.push(responseData.Id)
```
Comment (meyumeapps) [inline action-processor/create-action.js]: you could change `responseData.Id` to `jobId`

File: controller/generate-controller.js [hunk 2]
```diff
@@ -331,6 +334,28 @@ async function tearDownRegistrationJobs(featureName) {
...
+        for (let internalRef of maRenewalJobsList) {
+            log.info(`${CONTROLLER_NAME}::about to teardown renew ma jobs with internal reference ${internalRef}`)
+            await jobService.deleteJob(internalRef)
+        }
+    }
+}
+
+async function tearDownNewMAJobs (featureName) {
+    log.info(`${CONTROLLER_NAME}::tearDownNewMAJobs:${featureName}`)
+    const maRenewJobsList = localStorage.getItem(featureName, 'maNewMAJobs')
```
Comment (meyumeapps) [inline controller/generate-controller.js]: `maRenewJobsList` => `maNewJobsList`

File: controller/generate-controller.js [hunk 3]
```diff
@@ -331,6 +334,28 @@ async function tearDownRegistrationJobs(featureName) {
...
+    if (maRenewJobsList) {
+        for (let internalRef of maRenewJobsList) {
```
Comment (meyumeapps) [inline controller/generate-controller.js]: ditto

File: controller/generate-controller.js (hunk 2 above)
Comment (UntappedSolutions) [reply controller/generate-controller.js]: change made

File: controller/generate-controller.js (hunk 3 above)
Comment (UntappedSolutions) [reply controller/generate-controller.js]: change made

File: action-processor/create-action.js (hunk 1 above)
Comment (UntappedSolutions) [reply action-processor/create-action.js]: changed to responseData.Identifier which the other use. Ty for pointing out

**Pull request merged**